app/plot_app/templates/header.html

```diff
@@ -56,7 +56,9 @@ <h1>
 			<div class="dropdown-menu">
 				<a class="dropdown-item" href="download?log={{ log_id }}">Log File</a>
 				<a class="dropdown-item" href="download?log={{ log_id }}&type=1" target="_blank">Parameters</a>
```

Comment:
I think we can remove this option (but keep the implementation), and just offer the QGC versions.